lua/lazyvim/plugins/extras/lang/ocaml.lua

```diff
@@ -0,0 +1,40 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "ml", "mli", "cmi", "cmo", "cmx", "cma", "cmxa", "cmxs", "cmt", "cmti", "opam" },
+      root = { "merlin.opam", "dune-project" },
+    })
+  end,
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "ocaml" })
+      end
+    end,
+  },
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        ocamllsp = {
+          get_language_id = function(_, ftype)
+            return ftype
+          end,
```

Comment:
Are you sure this is correct? In [nvim-lspconfig ocamllsp docs](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/ocamllsp.lua) it seems to return the `language_id` of the `ftype`, not the `ftype` itself. Is this actually needed?

Reply:
Good catch!